src/Avalonia.Controls/DropDown.cs

```diff
@@ -114,6 +114,30 @@ protected override void OnKeyDown(KeyEventArgs e)
                     IsDropDownOpen = false;
                     e.Handled = true;
                 }
+
+                if (!IsDropDownOpen)
+                {
+                    if (e.Key == Key.Right)
+                    {
+                        if (++SelectedIndex >= ItemCount)
+                            SelectedIndex = 0;
+                        
+                        e.Handled = true;
+                    }
+                    else if (e.Key == Key.Left)
+                    {
+                        if (--SelectedIndex < 0)
+                            SelectedIndex = ItemCount - 1;
+                        e.Handled = true;
+                    }
+                    else if (e.Key == Key.Down)
+                    {
+                        IsDropDownOpen = true;
+                        if (SelectedIndex == -1)
+                            SelectedIndex = 0;
+                        e.Handled = true;
+                    }
+                }
```

Comment:
This doesn't seem to be the correct behavior (on windows at least) - on windows the up/down arrows, when the dropdown isn't open, cycle through the available items. It's Alt+Down that opens the dropdown.

Reply:
I've made the changes however the keydown event for up/down don't appear to propagate to the dropdown, I'm not sure why that happens. As result the changes do not currently work.